conf/nc24s_v3.conf

```diff
@@ -23,8 +23,7 @@
  * || check_hw_swap 0kB 0kB 3%
  * || check_hw_eth eth0
  * || check_hw_eth lo
- * || check_hw_eth docker0
-
```

Comment:
Do you want to get rid of this space?

Reply:
Most likely not for the sake of formatting

Reply:
addressed